spt/features/aim.cpp

```diff
@@ -177,6 +205,58 @@ CON_COMMAND(tas_aim, "Aims at a point.")
 	}
 }
 
+CON_COMMAND(tas_aim_pos, "Aims at a position. Usage: tas_aim_pos <x> <y> <z> [ticks]")
+{
+	if (args.ArgC() < 4)
+	{
+		Msg("Usage: tas_aim_pos <x> <y> <z>\n");
+		return;
+	}
+
+	int frames = -1;
+	if (args.ArgC() == 5)
+		frames = std::atoi(args.Arg(4));
+
+	spt_aim.viewState.state = aim::ViewState::AimState::POSITION;
+	spt_aim.viewState.targetPos = Vector(std::atof(args.Arg(1)), std::atof(args.Arg(2)), std::atof(args.Arg(3)));
+
+	if (frames == -1)
+	{
+		spt_aim.viewState.timedChange = false;
+	}
+	else
+	{
+		spt_aim.viewState.timedChange = true;
+		spt_aim.viewState.ticksLeft = std::max(1, frames);
+	}
+}
+
+CON_COMMAND(tas_aim_ent, "Aims at a entity. Usage: tas_aim_ent <id> [ticks]")
```

Comment:
this might be more useful in hl2 if you could aim at entity + offset, just aiming at abs origin would aim at an NPC's feet for example